messaging/src/main/java/org/axonframework/commandhandling/distributed/DistributedCommandBus.java

```diff
@@ -35,6 +35,10 @@
 import org.axonframework.messaging.MessageHandlerInterceptor;
 import org.axonframework.monitoring.MessageMonitor;
 import org.axonframework.monitoring.NoOpMessageMonitor;
+import org.axonframework.tracing.NoOpSpanFactory;
```

Comment:
Nit: I would expect the copyright notice to be updated.

Reply:
changed